weave/integrations/notdiamond/cassettes/tracing_test/test_notdiamond_quickstart.yaml

```diff
@@ -0,0 +1,4602 @@
+interactions:
+- request:
+    body: '{"query": "query DefaultEntity {\n  viewer {\n    username\n    defaultEntity
+      {\n      name\n    }\n  }\n}"}'
+    headers:
+      Accept:
+      - '*/*'
+      Accept-Encoding:
+      - gzip, deflate, zstd
+      Connection:
+      - keep-alive
+      Content-Length:
+      - '108'
+      Content-Type:
+      - application/json
+      User-Agent:
+      - python-requests/2.32.3
+    method: POST
+    uri: https://api.wandb.ai/graphql
+  response:
+    body:
+      string: '{"data":{"viewer":{"username":"a9_nd","defaultEntity":{"name":"notdiamond"}}}}'
+    headers:
+      Alt-Svc:
+      - h3=":443"; ma=2592000,h3-29=":443"; ma=2592000
+      Content-Encoding:
+      - gzip
+      Content-Type:
+      - application/json; charset=utf-8
+      Date:
+      - Tue, 01 Oct 2024 01:04:10 GMT
+      Transfer-Encoding:
+      - chunked
+      Vary:
+      - Origin
+      Via:
+      - 1.1 google
+      X-Content-Type-Options:
+      - nosniff
+      X-Ratelimit-Limit:
+      - '3000'
+      X-Ratelimit-Remaining:
+      - '2999'
+    status:
+      code: 200
+      message: OK
+- request:
+    body: '{"query": "query ProjectDetails($entity: String, $project: String) {\n  model(name:
+      $project, entityName: $entity) {\n    id\n    name\n    repo\n    dockerImage\n    description\n  }\n}\n",
+      "variables": {"entity": "notdiamond", "project": "dev_testing"}}'
+    headers:
+      Accept:
+      - '*/*'
+      Accept-Encoding:
+      - gzip, deflate, zstd
+      Connection:
+      - keep-alive
+      Content-Length:
+      - '255'
+      Content-Type:
+      - application/json
+      User-Agent:
+      - W&B Internal Client 0.18.2
+    method: POST
+    uri: https://api.wandb.ai/graphql
+  response:
+    body:
+      string: '{"data":{"model":{"id":"UHJvamVjdDp2MTpkZXZfdGVzdGluZzpub3RkaWFtb25k","name":"dev_testing","repo":null,"dockerImage":null,"description":null}}}'
+    headers:
+      Alt-Svc:
+      - h3=":443"; ma=2592000,h3-29=":443"; ma=2592000
+      Content-Encoding:
+      - gzip
+      Content-Type:
+      - application/json; charset=utf-8
+      Date:
+      - Tue, 01 Oct 2024 01:04:10 GMT
+      Transfer-Encoding:
+      - chunked
+      Vary:
+      - Origin
+      Via:
+      - 1.1 google
+      X-Content-Type-Options:
+      - nosniff
+      X-Ratelimit-Limit:
+      - '3000'
+      X-Ratelimit-Remaining:
+      - '2999'
+    status:
+      code: 200
+      message: OK
+- request:
+    body: null
+    headers:
+      Accept:
+      - '*/*'
+      Accept-Encoding:
+      - gzip, deflate, zstd
+      Connection:
+      - keep-alive
+      User-Agent:
+      - python-requests/2.32.3
+    method: GET
+    uri: https://raw.githubusercontent.com/BerriAI/litellm/main/model_prices_and_context_window.json
```

Comment:
This is sort of an odd entry that gets added when you have `mistralai` installed. It is not needed for tests (typically) and can be safely removed to save space